edx_exams/apps/api/v1/views.py

```diff
@@ -729,38 +730,44 @@ def get(self, request, course_id, content_id):  # pylint: disable=unused-argumen
         return Response(data)
 
 
-class CourseProviderSettingsView(ExamsAPIView):
+class ProctoringSettingsView(ExamsAPIView):
     """
-    Endpoint for getting provider related settings.
+    Endpoint for getting an exam's proctoring settings, including course-wide settings like the escalation email and
+    exam-specific settings like provider settings.
 
     exam/provider_settings/course_id/{course_id}/exam_id/{exam_id}
 
     Supports:
         HTTP GET:
-            Returns provider specific information given an exam_id
+            Returns exam configuration settings given an exam_id
 
             {
                 provider_tech_support_email: '',
                 provider_tech_support_phone: '',
                 provider_name: 'test provider',
+                escalation_name: 'test@example.com',
             }
     """
 
     authentication_classes = (JwtAuthentication,)
     permission_classes = (IsAuthenticated,)
 
-    def get(self, request, course_id, exam_id):  # pylint: disable=unused-argument
+    def get(self, request, course_id, exam_id):
         """
-        HTTP GET handler. Returns provider specific information given an exam_id
+        HTTP GET handler. Returns exam configuration settings given an exam_id
 
-        The course_id is provided as a path parameter to account for the exams middleware
+        The course_id is provided as a path parameter to account for the exams ExamRequestMiddleware router.
         """
         provider = get_provider_by_exam_id(exam_id)
+        config_data = get_course_exam_configuration_by_course_id(course_id)
```

Comment:
I went back and forth about whether to query this by `exam_id` or `course_id`. I went with `course_id`, because it's analogous to the `CourseExamConfiguration.get_configuration_for_course` method. My concern here was the unlikely case that there's a misconfiguration and the `course_id` provided is not the same as the `exam.course_id`, and querying a `CourseExamConfiguration` by `course_id` and by `exam_id` returns different configurations. Unlikely, but something that is an edge-case.